web/phpunit.xml

```diff
@@ -15,7 +15,7 @@
     <env name="BROWSERTEST_OUTPUT_DIRECTORY" value="/app/web/sites/simpletest/browser_output" />
     <env name="SIMPLETEST_BASE_URL" value="http://nginx:8080" />
     <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled" />
-    <env name="MINK_DRIVER_ARGS_WEBDRIVER" value='["chrome", null, "http://webdriver:4444/wd/hub"]' />
+    <env name="MINK_DRIVER_ARGS_WEBDRIVER" value='["chrome", {"browserName":"chrome","chromeOptions":{"args":["--disable-gpu","--headless", "--no-sandbox", "--disable-dev-shm-usage"]}}, "http://webdriver:9515"]' />
```

Comment:
We should rather define these in container environment variables - just like here: https://github.com/Pronovix/devportal-drupal-module/blob/8.x-2.x/.travis/docker-compose.yml#L33 and get rid of our custom phpunit.xml. We should re-use what Drupal core offers because that is always up-to-date.

Reply:
After that here is how PHPUnit should be executed: https://github.com/Pronovix/devportal-drupal-module/blob/8.x-2.x/.travis/run-test.sh#L19